packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -60,7 +62,7 @@ export const END_CALL_PAGES: CallCompositePage[] = [
 ];
 
 /**
- * {@link CallAdapter} state for pure UI purposes.
+ * {@link CallAdapterCommon} state for pure UI purposes.
```

Comment:
CallAdapter is used for ACS already, if we move it to AzureCommunicationCallAdapter, it would be a breaking type change :(
What do you think of CommonCallAdapter? Or AbstractCallAdapter?

Reply:
We chose CommonCallAdapter to align with handler layer in the end